pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/ParticipantDao.java

```diff
@@ -45,6 +50,13 @@
                 .findFormResponsesWithAnswersByUserIds(studyId, participants.keySet())
                 .forEach(resp -> participants.get(resp.getParticipantId()).addResponse(resp));
 
+        PexInterpreter pexInterpreter = new TreeWalkInterpreter();
```

Comment:
I wonder if we can compute the date-of-majority _outside_ of this class? So we don't have to `new` up an interpreter and find the study's policy in here (these two things doesn't seem "related" to "find participants").